src/js/background.js

```diff
@@ -28,16 +26,113 @@ var loading = false;
 var intervalId;
 var iconIndex = 0;
 
-function startRecording(id) {
-  tabId = id;
+let blobList = []
+let imageList = []
 
-  startScreenRecording(id);
-  startConsoleRecording(id);
-  startNetworkRecording(id);
+let logs = {}
+let filtered = {
+  console: [],
+  network: []
 }
 
+// for screent shot
+var screenshot = {
+  content: document.createElement("canvas"),
+  data: '',
+
+  init: function () {
+    this.initEvents();
+  },
+
+  saveScreenshot: function (tabs) {
+    var image = new Image();
+    image.onload = function () {
+      var canvas = screenshot.content;
+      canvas.width = image.width;
+      canvas.height = image.height;
+      var context = canvas.getContext("2d");
+      context.drawImage(image, 0, 0);
+
+      // save the image
+      var link = document.createElement('a');
+      // link.href=window.location.href
+      let title = ''
+      if (tabs != undefined & tabs[0] != undefined)
+        title = tabs[0].title
+      let screenshotTime = new Date().toString()
+      link.download = title + '_' + screenshotTime + ".png";
+
+      link.href = screenshot.content.toDataURL();
+
+      canvas.toBlob(function (blob) {
```

Comment:
save image details in local storage after successful conversion of blob

Reply:
canvas removed